### ISSUE TITLE
Autosaved, empty document shows no content after quitting and reopening Compositor

### ISSUE THREAD

**Issue by Dirk-** [ktraunmueller/Compositor#549]
Compositor 1.17, macOS 11.2.3:

- Open Compositor
- Select "New...", "Standard Article", "Choose"
- Untitled document is shown:

<img width="800" alt="Bildschirmfoto 2021-04-09 um 11 45 27" src="https://user-images.githubusercontent.com/7225580/114162718-a8d25e80-9929-11eb-88ea-74ff21043c87.png">

- Quit Compositor (option "Close windows when quitting applications" is not selected in System Preferences)
- Start Compositor
- Untitled document is shown without a page to edit:

<img width="800" alt="Bildschirmfoto 2021-04-09 um 11 46 26" src="https://user-images.githubusercontent.com/7225580/114162519-76286600-9929-11eb-9dfb-04dd2a86ad8d.png">

**Comment by ktraunmueller** [comment]
Yes, I noticed this problem a while ago #327. Time to look into this again.

Thanks for the very nice bug report 👍

**Comment by Dirk-** [comment]
You are welcome. Thanks for the nice tool!
